icaruscode/TPC/Tracking/cluster3D/SnippetHit3DBuilderICARUS_tool.cc

```diff
@@ -1660,8 +1655,8 @@ geo::WireID SnippetHit3DBuilderICARUS::NearestWireID(const Eigen::Vector3f& posi
         mf::LogWarning("SnippetHit3D") << "Exception caught finding nearest wire, position " << std::endl;
 
         // Assume extremum for wire number depending on z coordinate
-        if (position[2] < 0.5 * m_geometry->DetLength()) wireID.Wire = 0;
```

Comment:
Not sure if these two lengths are the same in the updated v10 interface/

Reply:
Yes, they are the same length.